pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -202,8 +202,10 @@ type PriorityQueue struct {
 }
 
 type inFlightPod struct {
-	pod              *framework.QueuedPodInfo
+	pod              *v1.Pod
 	moveRequestCycle int64
+	// successRequeuePlugins has the name of Requeue plugins which return success for the event happen during flight.
+	successRequeuePlugins sets.Set[string]
```

Comment:
This would be a good place to explain not just what this field has, but also it means. What is "the event" here? There may be multiple different events while the pod is in flight.